style2.css

```
@@ -0,0 +1,72 @@
+
+body {
+    font-family: "Roboto", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    text-transform: uppercase;
+}
+
+.container {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    align-items: center;
+    text-align: center;
+    position: absolute;
+    width: 1200px;
+    height: 60px;
+    left: 0px;
+    top: 0px;
+    list-style: none;
+    background: #FFFFFF;
+    box-shadow: 0px 2px 4px rgba(0, 0, 0, 0.05); 
+}
+
+.box {
+   display: flex;
+   margin: 10px;
+   padding-top: 20px;
+   padding-bottom: 20px;
```

```suggestion
   justify-content: center;
   align-items: center;
```
Якщо ти використовуєш ці падінги для центрування, то краще так не робити. Тут display: flex. Отже, можна центрувати таким чином